assets/base.css

```diff
@@ -177,13 +177,15 @@ html.no-js .no-js-hidden {
   padding: 0 1.5rem;
 }
 
-.page-width-desktop {
+.page-width-desktop,
+.page-width-tablet {
```

Comment:
It seems like there is really one breakpoint where `page-width-tablet` is different. It feels like there could be a simpler way to add it 🤔 

Like checking here differently: `{% if posts_displayed < 3 %}page-width-tablet{% else %}page-width-desktop{% endif %}` vs `page-width-desktop{% if posts_displayed < 3 %} page-width-tablet{% endif %}`